compiler/rustc_hir_typeck/src/upvar.rs

```diff
@@ -908,19 +909,13 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
     /// Combines all the reasons for 2229 migrations
     fn compute_2229_migrations_reasons(
```

Comment:
:thinking: should we just remove this function :sweat: we're now simply creating a struct